internal/pkg/manifest/svc.go

```diff
@@ -628,7 +632,8 @@ func (s *BackendService) HTTPLoadBalancerTarget() (targetContainer string, targe
 	return
 }
 
-func httpLoadBalancerTarget(exposedPorts ExposedPortsIndex, rrTargetPort *uint16) (targetContainer *string, targetPort *string) {
+// targetContainerFromTargetPort returns target container and target port from the given target_port input.
+func targetContainerFromTargetPort(exposedPorts ExposedPortsIndex, rrTargetPort *uint16) (targetContainer *string, targetPort *string) {
```

Comment:
nit: I supposed `rr` standed for routing rule? 
```suggestion
func targetContainerFromTargetPort(exposedPorts ExposedPortsIndex, targetPort *uint16) (targetContainer *string, targetPort *string) {
```